src/components/SystemsViewTable/SystemsViewFilters.tsx

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import {
+  DataViewCheckboxFilter,
+  DataViewTextFilter,
+} from '@patternfly/react-data-view';
+import DataViewFilters from '@patternfly/react-data-view/dist/cjs/DataViewFilters';
+import {
+  ApiHostGetHostListRegisteredWithEnum,
+  ApiHostGetHostListStalenessEnum,
+} from '@redhat-cloud-services/host-inventory-client/ApiHostGetHostList';
+
+export interface InventoryFilters {
+  name: string;
+  status: ApiHostGetHostListStalenessEnum[];
+  dataCollector: ApiHostGetHostListRegisteredWithEnum[];
+  rhcStatus: string[];
+  systemType: string[];
+}
+interface SystemsViewFiltersProps {
+  filters: InventoryFilters;
+  // eslint-disable-next-line no-unused-vars
+  onSetFilters: (_: Partial<InventoryFilters>) => void;
+}
+
+export const SystemsViewFilters: React.FC<SystemsViewFiltersProps> = ({
+  filters,
+  onSetFilters,
+}) => (
+  <DataViewFilters
+    onChange={(_, values) => onSetFilters(values)}
+    values={filters}
+  >
+    <DataViewTextFilter
+      filterId="name"
+      title="Name"
+      placeholder="Filter by name"
+    />
+    <DataViewCheckboxFilter
+      filterId="status"
+      title="Status"
+      placeholder="Filter by status"
+      options={[
```

Comment:
I'm not sure here, but it feels weird to repeat the values from the enums (here ApiHostGetHostListRegisteredWithEnum).

Do you think we could make it less error prone somehow?
Admittedly I'm not really sure how, the best I have is a function generating these options.
I'm happy to be told we'll do this as a follow-up.